frontend/public/components/utils/build-hooks.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+import * as _ from 'lodash-es';
+
+import { K8sResourceKind } from '../../module/k8s';
+
+export const BuildHooks: React.SFC<BuildHooksProps> = ({ resource }) => {
+  const postCommitArgs = _.get(resource, 'spec.postCommit.args');
+  const postCommitCommand = _.get(resource, 'spec.postCommit.command');
+  const postCommitScript = _.get(resource, 'spec.postCommit.script');
+
+  return (postCommitArgs || postCommitCommand || postCommitScript)
```

Comment:
Since two of these are arrays, probably better to use

```js
!_.isEmpty(postCommitCommand) || !_.isEmpty(postCommitArgs) || postCommitScript
```

That way there's no danger of showing an empty array